docs/changelog.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Append generated elements by `SCs-helper` into requested output structure
 - Oriented sets representation (<a, ..., b>) in SCs
+- Add output structure builder feature to config
```

Comment:
Move it into the begin of this section, please. The first messages of each section show what was changed last, and the last messages show what was done the firstly